spec/requests/fitness_logs_spec.rb

```diff
@@ -15,5 +15,34 @@
      expect(response).to have_http_status(200)
```

Comment:
Sorry, one more thing...should add tests for the index on each column too.  After that you should be good!